### PR TITLE
Fix #13774 - Tab navigation broken on p-dropdown

### LINKED ISSUE #13774: tab navigation broken on p-dropdown
### Describe the bug

Since 16.3.X version, navigation with tabulation key is broken by p-dropdown component
image

You can try  here it's KO:
https://primeng.org/dropdown

It's OK on 16.2.X and older version:
https://www.primefaces.org/primeng-v15-lts/dropdown

### Environment

prime ng 16.3.X and newer

### Reproducer

_No response_

### Angular version

16.4.X

### PrimeNG version

16.X

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.14

### Browser(s)

chrome

### Steps to reproduce the behavior

- click on the page
- navigate with tab key

when focus comes to p-dropdown, tab key lost navigation

### Expected behavior

when focus comes to p-dropdown, and after another tab keypress, focus must going to next element

### PR DESCRIPTION
**Pull request by stephanebouget**
Fix #13774 

With that fix, keyboard navigation with "tab" key works on dropdown component

### CODE REVIEW

File: src/app/components/dropdown/dropdown.ts [hunk 1]
```diff
@@ -1308,7 +1308,6 @@ export class Dropdown implements OnInit, AfterViewInit, AfterContentInit, AfterV
             case 27:
             case 9:
                 this.hide();
-                event.preventDefault();
```
[the remark this answers is not in the document]
Comment (SoyDiego) [reply src/app/components/dropdown/dropdown.ts:1311]: Like I said, someone add this to fix another problem:

> When closing the dropdown overlay via 'Escape' key the event should also be default prevented to give parent elements a chance to detect that the event was already processed

Here you have the commit: https://github.com/primefaces/primeng/commit/b1efe18a85b7e0bb86ecbf93fe830ad2b67f86ff.

Another solution to one of this fixes and mantain the other, I don't know.
Maybe @JanST123 who was introduced this line can help too.

**Comment by JanST123** [comment]
@SoyDiego @stephanebouget My solution would have been to only preventDefault for the 'Escape' key, not for 'Tab'. But having a look at the current state of dropdown.ts in the master, I said this thing was completely refactored in f52340b.

Having a look at the new code, the issue should already been resolved.

**Comment by SoyDiego** [comment]
> @SoyDiego @stephanebouget My solution would have been to only preventDefault for the 'Escape' key, not for 'Tab'. But having a look at the current state of dropdown.ts in the master, I see this thing was completely refactored in [f52340b](https://github.com/primefaces/primeng/commit/f52340b9daf5ea2ee921fc0906838b98007cd6ed).
> 
> Having a look at the new code, the issue should already been resolved.

Thanks @JanST123, yes maybe now it's solved because this branch has conflicts and is not updated.

**Comment by stephanebouget** [comment]
Yes, i confirm that in the current release version (16.6.0) the dropdown can be closed with "escape" key without need preventing default event. So for me, the MR can be merged.

**Comment by SoyDiego** [comment]
> Yes, i confirm that in the current release version (16.6.0) the dropdown can be closed with "escape" key without need preventing default event. So for me, the PR can be merged.

This PR can be merged?
You have some conflicts 
image

**Comment by stephanebouget** [comment]
> > Yes, i confirm that in the current release version (16.6.0) the dropdown can be closed with "escape" key without need preventing default event. So for me, the PR can be merged.
> 
> This PR can be merged? You have some conflicts image

This component has been refactored into the fresh 16.7.0 and the issue no longer appears. So this PR can be closed.

**Comment by SoyDiego** [comment]
> > > Yes, i confirm that in the current release version (16.6.0) the dropdown can be closed with "escape" key without need preventing default event. So for me, the PR can be merged.
> > 
> > 
> > This PR can be merged? You have some conflicts image
> 
> This component has been refactored into the fresh 16.7.0 and the issue no longer appears. So this PR can be closed.

Great thanks, you can closed the PR because it's yours :)

**Pull request closed without merging**